docs/interfaces/c61/modem.md

```diff
@@ -0,0 +1,93 @@
+---
+title: C61 Modem
+tags:
+    - C61
+    - CT GL
+---
+
+## Overview 
+
+The C61 has a modem from Quectel called EG25-G with a GPS receiver.
+
+## Starting/Enabling
+
+**Note:**  The modem is turned off by default.
+```
+echo 1 > /opt/hm/pic_attributes/ctrl_modem_on
+```
+* The modem takes about 10 seconds to start and register itself with the kernel. And thakes up to one minutes before it is registered in the linux usb list.
+
+* Check USB TTY device to see if it has shown up.
+
+
+```bash
+find /dev -name "ttyUSB*"
+```
+
+```bash
+/dev/ttyUSB3
+/dev/ttyUSB2
+/dev/ttyUSB1
+/dev/ttyUSB0
+```
+
+* The modem also shows up with the `lsusb` command.
+
+```bash
+root@10210100023:~# lsusb | grep Quectel
+Bus 001 Device 007: ID 2c7c:030a Quectel Wireless Solutions Co., Ltd. Quectel EM05-G
+```
+
+# Status ON
```

Comment:
```suggestion
# Status (ON/OFF)
```

Reply:
it is the status on status off is not showing as an example

Reply:
OK, good. Then maybe clarify that